classes/testing_idealist.rb

```diff
@@ -61,12 +61,12 @@ def estimate_testingness_in_file file_path
       return has_no_tests
     end
     
-    regexes = [/XCTAssert|XCTFail/,        # XCTest
-               /expect\(/,                 # Expecta, Nimble
-               /should\]|shouldNot\]/,     # Kiwi
-               /assertThat/,               # OCHamcrest
-               / should .*;| should_not /, # Cedar
-               /FBSnapshotVerify/          # FBSnapshotTestCase
+    regexes = [/XCTAssert|XCTFail/,                 # XCTest
+               /expect\(/,                          # Expecta, Nimble
+               /should\]|shouldNot\]/,              # Kiwi
+               /assertThat/,                        # OCHamcrest
+               / should .*;| should_not |expect\(/, # Cedar
```

Comment:
This current implementation finds the count of each of the regexes in the list, and returns the highest count. It would need some adjustment to support "mixed" expectation types in a single file.